src/util/ses.ts

```diff
@@ -908,29 +908,34 @@ export function sendNFTBookSalesEmail({
   return ses.sendEmail(params).promise();
 }
 
-export function sendNFTBookSaleCommissionEmail({
+export function sendNFTBookSaleCommissionsEmail({
```

Comment:
I think `commission` is a little misleading here, I suggest using `payment`
```suggestion
export function sendNFTBookSalePaymentsEmail({
```